src/ucp/core/ucp_ep.c

```diff
@@ -1201,6 +1241,11 @@ static void ucp_ep_discard_lanes(ucp_ep_h ep, ucs_status_t discard_status)
         }
 
         ucs_debug("ep %p: discard uct_ep[%d]=%p", ep, lane, uct_ep);
+
+        if (ucp_ep_can_unprogress_lane(ep, uct_ep)) {
+            discard_arg->unprogress_mask |= UCS_BIT(lane);
+        }
+
```

Comment:
can we move this inside ucp_worker_discard_tl_uct_ep?
then, need to delete ucp_wireup_ep_aux_ep_discarded, and will not need special case for wireup_ep since unprogress will be done by the common function ucp_worker_discard_tl_uct_ep

Reply:
done